models/category.js

```diff
@@ -16,13 +16,31 @@ module.exports = (sequelize, DataTypes) => {
     {
       Name: DataTypes.STRING,
       IconBackImg: DataTypes.STRING,
+      IconBackImgUrl: {
+        type: DataTypes.VIRTUAL,
+        get() {
+          return this.assetUrl('IconBackImg');
+        },
+      },
       NavBackImg: DataTypes.STRING,
+      NavBackImgUrl: {
+        type: DataTypes.VIRTUAL,
+        get() {
+          return this.assetUrl('NavBackImg');
+        },
+      },
       Position: DataTypes.INTEGER,
     },
     {
       sequelize,
       modelName: 'Category',
     }
   );
+
+  Category.afterSave(async (category, options) => {
+    category.handleAssetFile('IconBackImg', options);
+    category.handleAssetFile('NavBackImg', options);
+  });
```

Comment:
This code will handle properly processing the file upload with the record is saved.